user_sync/rules.py

```diff
@@ -502,6 +502,9 @@ def process_strays(self, umapi_connectors):
             self.write_stray_key_map()
         if self.will_manage_strays:
             max_missing = self.options['max_adobe_only_users']
+            if isinstance(max_missing, str) and '%' in max_missing:
+                percent = float(max_missing.strip('%'))/100
+                max_missing = int((self.primary_user_count - self.excluded_user_count) * percent)
             if stray_count > max_missing:
                 self.logger.critical('Unable to process Adobe-only users, as their count (%s) is larger '
                                      'than the max_adobe_only_users setting (%d)', stray_count, max_missing)
```

Comment:
If a percentage is in use, can put that value in the parenthesis instead of the value of `max_missing`?  

Example:

```
Unable to process Adobe-only users, as their count (5) is larger than the max_adobe_only_users setting (10%)
```